### PR TITLE
Fm/unique barcode generation

### PR DESCRIPTION
**Pull request by FahimehMirhaj**

### CODE REVIEW

File: barcode-backend/services/services.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      padded = true;
+    }
+    let uniquePlateBarcode = String(plateType) + "_" + String(year) + newCounter;
+    listOfBarcodes.push(uniquePlateBarcode);
+    await barcodModel.findOneAndUpdate({plateType: plateType},{$set: { counter: newCounter }}); 
+  }
+  return listOfBarcodes;
+};
+
+exports.getCatFact = async function () {
```
Comment (sdelbelso) [inline barcode-backend/services/services.js]: Can we remove this unused code

File: barcode-backend/controllers/PlateBarcodesController.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        // The parameter is missing, example response...
+        res.status(400).json({ error: 'missing parameter count', data: null });
+        return;
+      }
+    return req.body.count;
+    // res.status(200).json({ error: null, data: req.body.count});
+  },
+];
+
+exports.getCatFact = [
```
Comment (sdelbelso) [inline barcode-backend/controllers/PlateBarcodesController.js]: Remove unused controller

File: barcode-frontend-vue/README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# template-frontend-vue
```
Comment (sdelbelso) [inline barcode-frontend-vue/README.md]: Can you please remove the entire barcode-frontend-vue folder

File: barcode-backend/controllers/PlateBarcodesController.js (hunk 2 above)
Comment (FahimehMirhaj) [reply barcode-backend/controllers/PlateBarcodesController.js]: Addressed.

File: barcode-frontend-vue/README.md (hunk 3 above)
Comment (FahimehMirhaj) [reply barcode-frontend-vue/README.md]: Addressed.

File: barcode-backend/services/services.js (hunk 1 above)
Comment (FahimehMirhaj) [reply barcode-backend/services/services.js]: Addressed.

**Review by sdelbelso — APPROVED** [review]

**Pull request merged**